README.md

```diff
@@ -3,3 +3,49 @@
 Spring conducted a phenotypic, high content imaging screen on inflammasome activated primary human PBMCs using a library of bioactive small molecules to identify novel inflammasome inhibitors. Using images from these phenotypic datasets, we established a computational method to score phenotypic similarity in order to identify compounds that are likely biologically related.
 
 This dataset is designed to benchmark the compound similarity method by allowing measurements of inherent sources of variability such as donors, plates, and experimental runs. We used a bespoke library of 623 compounds, profiled in 4 batches of 8 plates each across a total of 16 donors.
+
+## Explore the data
+
+The data can be visualized and analyzed in the Spring platform at https://app.springscience.com/explore-iinflammasome-benchmark.
+
+## Download the data
```

Comment:
With every dataset or data download it is important to provide the hash values to make sure users got the right data and it has not been tampered with or incomplete. GCS will give you the MD5 and CRC32 if you see the details tab.